src/main.cpp

```diff
@@ -153,71 +152,22 @@ int main(int argc, char* argv[]) {
 
     };
 
-    //compileRecursive("I:/ACE3/addons");
-    //compileRecursive("I:/CBA_A3/addons");
-    compileRecursive("T:/x/");
-    compileRecursive("T:/z/ace/");
-    compileRecursive("T:/z/acex/");
-    compileRecursive("T:/a3/");
-    //compileRecursive("P:/test/");
-
-    
-    //compileRecursive("I:/CBA_A3/addons");
-    //compileRecursive("I:/ACE3/addons/nightvision");
-    //compileRecursive("I:/ACE3/addons/aircraft");
-    workWait.unlock();
-
-
+    for (std::string &inputDir : inputDirs) {
+        compileRecursive(inputDir);
+    }
 
+    workWait.unlock();
 
-    std::unique_ptr<std::thread> myThread = std::make_unique<std::thread>(workerFunc);
-    std::unique_ptr<std::thread> myThread2 = std::make_unique<std::thread>(workerFunc);
-    std::unique_ptr<std::thread> myThread3 = std::make_unique<std::thread>(workerFunc);
-    std::unique_ptr<std::thread> myThread4 = std::make_unique<std::thread>(workerFunc);
-    std::unique_ptr<std::thread> myThread5 = std::make_unique<std::thread>(workerFunc);
-    std::unique_ptr<std::thread> myThread6 = std::make_unique<std::thread>(workerFunc);
-    std::unique_ptr<std::thread> myThread7 = std::make_unique<std::thread>(workerFunc);
+    std::vector<std::thread> workerThreads;
+    for (int i = 0; i < numberOfWorkerThreads; i++) {
+        workerThreads.push_back(std::thread(workerFunc));
+    }
 
     workerFunc();
 
-    myThread->join();
-    myThread2->join();
-    myThread3->join();
-    myThread4->join();
-    myThread5->join();
-    myThread6->join();
-    myThread7->join();
-
-    /*
-    auto compiledScript = compiler.compileScript("I:/ACE3/addons/advanced_ballistics/functions/fnc_readWeaponDataFromConfig.sqf");
```

Comment:
Thats my testing code :cry: I'd prefer to have this here so I can easily uncomment it for testing

Reply:
I'll restore it 🤣